composer.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "modx/revolution",
+  "description": "MODX Revolution is a Content Management System",
+  "type": "project",
+  "keywords": [
+    "content",
+    "CMS",
+    "routing",
+    "templating"
+  ],
+  "homepage": "http://modx.com/",
+  "license": "GPL-2.0+",
+  "authors": [
+    {
+      "name": "Jason Coward",
+      "email": "jason@opengeek.com",
+      "homepage": "http://jasoncoward.com/",
+      "role": "Developer"
+    },
+    {
+      "name": "Shaun McCormick",
+      "email": "splittingred@gmail.com",
+      "homepage": "http://splittingred.com/",
+      "role": "Developer"
+    },
+    {
+      "name": "Jan Peca",
+      "email": "pecajan@gmail.com",
+      "homepage": "http://www.bxr.cz/",
+      "role": "Developer"
+    }
+  ],
+  "support": {
+    "docs": "http://rtfm.modx.com/revolution/2.x/",
+    "email": "help@modx.com",
+    "forum": "http://forums.modx.com/",
+    "irc": "irc://irc.freenode.org/modx",
+    "issues": "https://github.com/modxcms/revolution/issues/",
+    "source": "https://github.com/modxcms/revolution/"
+  },
+  "config": {
+    "vendor-dir": "core/vendor"
+  },
+  "require": {
+    "php": ">=5.6",
+    "xpdo/xpdo": "^3.0@dev",
```

Comment:
As the PR targets 2.x, it should not include the 3.x dependencies (in both the require and autoload section)